CMSgov/design-system
# [WNMGDS-1836] Fix auto-scrolling to code snippets on load

## Summary

https://jira.cms.gov/browse/WNMGDS-1836

### Fixed

- Fixed unwanted scrolling to code snippets upon first loading a component page

## How to test

1. Go to our [deployed beta site](https://design.cms.gov/v/4.0.0-beta.2/components/accordion?theme=core) or build master locally
2. Click on the Components > Icon page
3. See that the page will scroll to the middle of the page.
4. Do the same with this branch to verify that it does not automatically scroll to the middle of the page

https://cmsgov.github.io/design-system/branch/pwolfert/fix-code-snippet-scrolling
